tools/conf.yaml

```
@@ -0,0 +1,11 @@
+bucket1:
+  bucket_name: "<bucket_name>"
+  credentials_file: "./credentials"
+  profile: "bucket1"
+  endpoint_url: "<endpoint_url>"
+
+bucket2:
+  bucket_name: "<bucket_name>"
+  credentials_file: "./credentials"
+  profile: "bucket2"
+  endpoint_url: "<endpoint_url>"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)